torchtext/csrc/core/vocabulary.h

```diff
@@ -0,0 +1,36 @@
+#ifndef VOCABULARY_H
+#define VOCABULARY_H
+
+#include <stdint.h>
```

Comment:
Why not use the cpp compatibility header https://en.cppreference.com/w/cpp/header/cstdint